src/jsifier.js

```diff
@@ -349,9 +348,6 @@ function ${name}(${args}) {
         }
       }
 
-      if (redirectedIdent) {
-        deps = deps.concat(LibraryManager.library[redirectedIdent + '__deps'] || []);
```

Comment:
This was previously dead code since `redirectedIdent` was always undefined prior to this change.  `deps` is declared as `const` which means this line would be runtime error if it ever ran.

The dependencies of the target are already handled by the  `deps.push(redirectedTarget);` above.